jcl/src/openj9.sharedclasses/share/classes/com/ibm/oti/shared/SharedClassAbstractHelper.java

```diff
@@ -223,8 +223,11 @@ public Integer run() {
 	 * @param filter the filter to use when finding and storing classes
 	 */
 	@Override
+	/*[IF JAVA_SPEC_VERSION < 24]*/
```

Comment:
```
import java.security.AccessController;
import java.security.PrivilegedAction;
```
L30-31 can be removed for JDK24+.

Reply:
The `AccessController.doPrivileged` use in this file is still there. I will remove this in a seperate pull request.